modules/globebrowsing/src/rawtiledatareader.cpp

```diff
@@ -439,6 +442,94 @@ RawTileDataReader::~RawTileDataReader() {
     }
 }
 
+std::optional<std::string> RawTileDataReader::getMRFCache() {
```

Comment:
Should be `mrfCache` since we only use `get` for functions that take in a reference or pointer, not returning the value in the return value